.vsts-ci.yml

```diff
@@ -3,14 +3,18 @@ trigger:
 - release/*
 
 variables:
-  teamName: Roslyn-Project-System
+  - name: teamName
+    value: Roslyn-Project-System
   ${{ if eq(variables['System.TeamProject'], 'public') }}:
-    PB_PublishBlobFeedKey:
-    PB_PublishBlobFeedUrl:
-    _DotNetPublishToBlobFeed: false
+    - name: PB_PublishBlobFeedUrl
```

Comment:
The conditions in this `variables` block have to be treated the same way as the rest of the values in the object.  ie, you have to treat them as array entries meaning prefix them with `- `

ie `- ${{ if eq(variables['System.TeamProject'], 'public') }}:`

Reply:
Done.